pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -388,6 +388,7 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 	p.updateCPUsToReuse(pod, container, cpuset)
 	p.updateMetricsOnAllocate(cpuset)
 
+	klog.V(4).InfoS("Allocated exclusive CPUs", "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name, "cpuset", "cpuset")
```

Comment:
It might be fine to have some log duplication but I'm just stating the fact the we're printing similar log at:
https://github.com/kubernetes/kubernetes/blob/cede96336a809a67546ca08df0748e4253ec270d/pkg/kubelet/cm/cpumanager/state/state_mem.go#L80

Reply:
The last variable should be `cpuset` and not `"cpuset"` (without quotes)

Reply:
Thanks, will update now.